### PR TITLE
Set flags on AWS CCM mimicking KCM

### PR DESCRIPTION
**Pull request by olemarkus**
With this PR, my local setup is down to 13 failures.

### CODE REVIEW

File: pkg/model/components/awscloucontrollermanager.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (rifelpet) [inline pkg/model/components/awscloucontrollermanager.go:1]: filename is missing the `d` in `cloud`

**Review by hakman — APPROVED** [review]

**Pull request merged**